StoreManagementService/src/PBJ.StoreManagementService.Models/Pagination/PaginationRequestModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PBJ.StoreManagementService.Models.Pagination
+{
+    public class PaginationRequestModel
+    {
+        public int Page { get; set; }
+
+        public int Take { get; set; }
+    }
+}
```

Comment:
Possible it is better to create specific request models derived from PaginationRequestModel. You definetly will need additional data inside.  for example **PostFilteredRequest : PaginationRequestModel** or something